xenon_interpreter.py

```diff
@@ -60,7 +60,6 @@ def parse(self):
   def execute(self):
     current_sweep = None
     for command in self.commands_:
-      print(command)
```

Comment:
IMO this seems like a genuinely useful debug statement.

Reply:
> IMO this seems like a genuinely useful debug statement.

Sure. Added it back.